packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -130,20 +130,13 @@ const ConnectorDetails = () => {
             <div>
               <ConnectorTypeName type={data.type} />
               <div className={styles.verticalBar} />
-              {data.type === ConnectorType.Social && inUse !== undefined && (
+              {inUse !== undefined && (
```

Comment:
nit: curious why not simply use `inUse` here?

Reply:
there is a "loading" state, not just true or false.